testdata/project-v4-multigroup/go.mod

```diff
@@ -3,7 +3,7 @@ module sigs.k8s.io/kubebuilder/testdata/project-v4-multigroup
 go 1.24.0
 
 require (
-	github.com/cert-manager/cert-manager v1.18.0
+	github.com/cert-manager/cert-manager v1.18.1
```

Comment:
Can you rebase with master?

Reply:
Yep, will do.

Reply:
I think we need to merge this first? #4875